src/content/docs/pl/concepts/why-astro.mdx

```diff
@@ -0,0 +1,80 @@
+5. [Pełny funkcji, ale elastyczny](#pełny-funkcji-ale-elastyczny): Ponad 100 integracji z Astro do wyboru.
+
+## Skoncentrowany na treści
+
+**Astro zostało zaprojektowane do budowania stron bogatych w treść.** Należą do nich większość stron marketingowych, wydawniczych, dokumentacje, blogi, portfolia oraz niektóre strony e-commerce.
+
+W przeciwieństwie do tego podejścia większość współczesnych webowych frameworków jest zaprojektowana w celu budowania *aplikacji internetowych*. Te frameworki sprawdzają się najlepiej przy tworzeniu bardziej zaawansowanych, przypominających aplikacje, doświadczeń: panele admina, skrzynki odbiorcze, serwisy społecznościowe, listy zadań, a nawet aplikacje podobne do natywnych jak [Figma](https://figma.com/) lub [Ping](https://ping.gg/).
+
+Jest to jedna z najważniejszych różnic, która pomaga zrozumieć Astro. Unikalny dla Astro nacisk na treść pozwala nam na podejmowanie decyzji, które pomagają dostarczyć niezrównaną wydajność oraz funkcjonalności, których nie miałoby sensu implementować we frameworkach służących do tworzenia aplikacji.
+
+:::tip
+Jeśli Twój projekt wpisuje się w kategorię "aplikacji", Astro może nie być właściwym wyborem... **i jest to w porządku!** Sprawdź [Next.js](https://next.org/), alternatywny do Astro framework do budowania aplikacji webowych.
+:::
+
+
+## Server-first
+
+**Astro korzysta z renderowania na serwerze (ang. *server-side*) zamiast na kliencie (ang. *client-side*) wszędzie, gdzie jest to możliwe**. Jest to rozwiązanie, które serwerowe frameworki — PHP, WordPress, Laravel, Ruby on Rails, itd. — stosują od dekad. Nie musicie jednak uczyć się osobnego języka, żeby korzystać z renderowania na serwerze. Z Astro, wszystko to wciąż tylko HTML, CSS i JavaScript (lub, jeśli wolicie, TypeScript).
+
+To podejście stoi w kontrze do innych współczesnych JavaScriptowych frameworków, takich jak Next.js, SvetleKit, Nuxt, Remix i podobnych. Wymagają one renderowania na kliencie całej strony i używają serwerowego renderowania głównie, by zaadresować problemy z wydajnością. Takie podejście zostało nazwane **Single Page App (SPA)**, w przeciwieństwie do podejścia **Multi Page App (MPA)**, które stosuje Astro.
+
+Model SPA ma swoje korzyści. Jednak są one okupione dodatkową złożonością i kompromisami związanymi z wydajnością. Te kompromisy obniżają wydajność strony — włączając w to istotne metryki, jak [Czas do Interaktywności (ang. *Time to Interactive* — TTI)](https://web.dev/interactive/) — i nie mają one większego sensu dla stron opartych na treści, gdzie szybkość pierwszego ładowania jest kluczowa.
+
+📚 [Dowiedz się więcej o tym, co czyni architekturę MPA Astro wyjątkową.](/pl/concepts/mpa-vs-spa/)
+
+
+## Domyślnie szybki
+
+Dobra wydajność zawsze jest ważna, lecz dla stron opartych o treści jest *wyjątkowo* krytyczna. To, że niska wydajność negatywnie wpływa na zaangażowanie, konwersje i traci pieniądze, zostało dobrze sprawdzone. Na przykład:
```

Comment:
```suggestion
Dobra wydajność zawsze jest ważna, lecz dla stron opartych o treści jest *wyjątkowo* krytyczna. To, że niska wydajność negatywnie wpływa na zaangażowanie, konwersje i potencjalny zysk, zostało dobrze sprawdzone. Na przykład:
```

To zdanie totalnie nie pasowało do całego kontekstu, w Angielskiej wersji widnieje ` well-proven that poor performance loses you engagement, conversions, and money.`